src/components/thumbnail-card/ThumbnailCardDetails.js

```diff
@@ -7,34 +7,36 @@ import { useIsContentOverflowed } from '../../utils/dom';
 type Props = {
     actionItem?: React.Element<any>,
     icon?: React.Node,
+    onKeyDownCallback?: () => void,
     subtitle?: React.Node,
     title: React.Node,
 };
 
 type TitleProps = {
+    onKeyDownCallback?: () => void,
     title: React.Node,
 };
 
-const Title = ({ title }: TitleProps) => {
-    // $FlowFixMe
+const Title = ({ title, onKeyDownCallback }: TitleProps) => {
     const textRef: { current: null | HTMLElement } = React.useRef(null);
     const isTextOverflowed = useIsContentOverflowed(textRef);
 
     return (
         <Tooltip isDisabled={!isTextOverflowed} text={title}>
-            <div ref={textRef} className="thumbnail-card-title">
+            {/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex, jsx-a11y/no-static-element-interactions */}
+            <div ref={textRef} role="link" className="thumbnail-card-title" tabIndex={0} onKeyDown={onKeyDownCallback}>
```

Comment:
Can this just be `onKeyDown` or do we need to keep the original onKeydown for this component?

Reply:
do you mean like `onKeyDown={onKeyDown}`?